scripts/test-llvm.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* @flow */
+
+import { prepackFileSync } from "../lib/prepack-node.js";
+import invariant from "../lib/invariant.js";
+
+let chalk = require("chalk");
```

Comment:
I don't know. This is copy-paste from the other test-serializer.

Ultimately, I'd like to run all the normal serializer tests through the LLVM path. At that point it might make sense to unify the runners.